popup.js

```diff
@@ -12,12 +12,31 @@ function sendRequest(data) {
 }
 
 jQuery(document).ready(function($) {
-    //send message request to content script
-    $('#btnUpload').click(function() {
-        sendRequest({
-            type: 'addJson',
-            jsontext: $('#jsontext').val()
-        });
+    $('#form-import').submit(function(e) {
```

Comment:
@jinhduong Because the form is handled from JS so I just added that attribute, it doesn't affect the form (it's mean you can remove or just ignore it).